inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -170,21 +170,17 @@ void GNADeviceHelper::releaseModel(const uint32_t model_id) {
 
 bool GNADeviceHelper::enforceLegacyCnnNeeded() const {
     const auto compileTargetDevice = getTargetDevice(false);
-    return isGnaLibVersion2_1 && isUpTo20HwGnaDevice(compileTargetDevice);
+    return (isGnaLibVersion3_0 || isGnaLibVersion2_1) && isUpTo20HwGnaDevice(compileTargetDevice);
```

Comment:
Should 2.1 stay here and below or do we need only 3.0?

Reply:
@sirzabek, @kbruniec asked to save this option